Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -1008,6 +1008,9 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Update Once-Conflicted Doc - SGCol
     SG::TestUser testUser { _sg, "uocd", chIDs, collectionSpecs };
     _sg.authHeader = testUser.authHeader();
 
+    _sg.assignUserChannel("sguser", { collectionSpecs.begin(), collectionSpecs.end() }, { chIDs });
+    _sg.authHeader = HTTPLogic::basicAuth("sguser", "password");
+
```

Comment:
Why are the above two lines?
"sguser" is a permanent user. My current thoughts are, we don't want to modify its channel access, because, if we do, we need to restore its original channel access after the test. Your TestUser is convenient class to use since the user would be automatically deleted after the test,

Reply:
You're right, this looks like what used to be in the test. I think this was added back during a merge conflict resolution.